src/analysis/Environments/Environments.test.ts

```diff
@@ -908,9 +909,12 @@ describe('Environments Component', () => {
       expect(props.onEvent).toBeCalledTimes(1);
       expect(props.onEvent).toBeCalledWith(
         'dataRefresh',
-        // times are zeroed out because of mocked data calls
-        { leoCallTimeMs: 0, totalCallTimeMs: 0, runtimes: 1, disks: 0, apps: 0 } satisfies DataRefreshInfo
+        expect.objectContaining({ runtimes: 1, disks: 0, apps: 0 } satisfies Partial<DataRefreshInfo>)
       );
+      const eventArgs = asMockedFn(props.onEvent as Required<typeof props>['onEvent']).mock
+        .calls[0][1] as DataRefreshInfo;
+      expect(typeof eventArgs.totalCallTimeMs).toBe('number');
+      expect(typeof eventArgs.leoCallTimeMs).toBe('number');
```

Comment:
I don't think we need tests for correct types. TypeScript should handle that.

```suggestion
```

Reply:
this wasn't so much about checking types, but letting the exact time value not fail the test as long as it was a valid number, since I was seeing the value be 1ms instead of 0ms.

Reply:
Ah. In that case, suggest...

```ts
      expect(props.onEvent).toBeCalledWith(
        'dataRefresh',
        { leoCallTimeMs: expect.any(Number), totalCallTimeMs: expect.any(Number), runtimes: 1, disks: 0, apps: 0 } satisfies DataRefreshInfo,
      );
```

Reply:
ooooh... yeah I like that.